widgets/heatmap/options.go

```diff
@@ -15,8 +15,9 @@
 package heatmap
```

Comment:
(nit) We may want to update the licence headers to say 2021 since it looks like it will land in that year.